2048/2048/src/main/res/values/strings.xml

```diff
@@ -52,4 +52,5 @@
   <string name="rankings">RANKINGS</string>
 
   <string name="entry_price_duel">1 USD</string>
+  <string name="geofencing_message"> Your game has ended because your region is not supported.</string>
```

Comment:
it seems that there's a white-space in the beginning of the strings

Reply:
also, this error message might not be super clear. for the end-user, might not be clear that a region can be unsupported. so, i would basically use the same error message that we have on the AppCoins wallet side: `Your game has ended because e-Skills games are not available in your location` (from: https://github.com/Catappult/appcoins-wallet-android/blob/master/skills/src/main/res/values/strings.xml#L18)

Reply:
changed it